src/Microsoft.ML.Data/Model/ModelOperationsCatalog.cs

```diff
@@ -135,6 +142,13 @@ private void SaveInputSchema(DataViewSchema inputSchema, RepositoryWriter rep)
         /// <param name="inputSchema">Will contain the input schema for the model. If the model was saved without
         /// any description of the input, there will be no input schema. In this case this can be <see langword="null"/>.</param>
         /// <returns>The loaded model.</returns>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[Load](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectChangePointBySsa.cs]
```

Comment:
>DetectChangePointBySsa [](start = 114, length = 22)

Same here. please use/create a sample with the bare minimum required to show the API usage.